conu/backend/origin/backend.py

```diff
@@ -174,8 +174,6 @@ def _create_app_from_template(self, image, name, template, name_in_template,
         except subprocess.CalledProcessError as ex:
             raise ConuException("oc new-app failed: %s" % ex)
 
-        self.start_build(name)
```

Comment:
I don't see why this has been removed.

Reply:
I realize that build was executed without explicitly call `oc start-build`.